requirements.txt

```diff
@@ -1,2 +1,3 @@
-annlite>=0.5.4
-docarray[annlite]>=0.16.4
\ No newline at end of file
+annlite==0.5.4
```

Comment:
are we sure about locking this annlite version ? Should not we adapt it to the new version ?